reana_workflow_controller/rest/workflows_workspace.py

```diff
@@ -344,9 +344,10 @@ def delete_file(workflow_id_or_name, file_name):  # noqa
             return jsonify({"message": "User {} does not exist".format(user)}), 404
 
         workflow = _get_workflow_with_uuid_or_name(workflow_id_or_name, user_uuid)
-        abs_path_to_workspace = os.path.join(
-            current_app.config["SHARED_VOLUME_PATH"], workflow.workspace_path
-        )
+        abs_path_to_workspace = workflow.workspace_path
```

Comment:
Similarly we could get rid of `abs_path_to_workspace` here as well. Same for other places